.changeset/many-penguins-hug.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/perseus": minor
```

Comment:
Since you changed PerseusTypes in a way that's not backwards compatible, I'd call this a major/breaking change.